infra/lib/fhir-server-stack.ts

```diff
@@ -53,8 +53,8 @@ export function settings(): Settings {
   if (isProd(config)) {
     return {
       ...defaults,
-      cpu: 2 * vCPU,
-      memoryLimitMiB: 4096,
+      cpu: 4 * vCPU,
+      memoryLimitMiB: 8192,
```

Comment:
Actually, considering the [original comment](https://github.com/metriport/fhir-server/pull/65/files#r1696169664) and the [weekly load of max CPU](https://us-west-1.console.aws.amazon.com/cloudwatch/home?region=us-west-1#metricsV2?graph=~(metrics~(~(~'AWS*2fECS~'CPUUtilization~'ServiceName~'FHIRServerStack-FHIRServerFargateService46BCF377-gXh9molzfoo4~'ClusterName~'FHIRServerStack-FHIRServerClusterC4EE8AC0-00Vo19v9bvc5~(region~'us-west-1))~(~'.~'MemoryUtilization~'.~'.~'.~'.~(region~'us-west-1~yAxis~'right)))~period~300~region~'us-west-1~stacked~false~title~'CPU*20utilization~view~'timeSeries~stat~'Maximum~start~'-PT168H~end~'P0D)), taking a closer look at some of the spikes, we see they run for ~100% for minutes, so its clear the CPU is also a bottleneck - see below:

Jul 27:

![image](https://github.com/user-attachments/assets/396befa7-8837-45dc-8f26-38a72eb4547a)

Jul 26

![image](https://github.com/user-attachments/assets/eefafd1c-c6eb-4d82-aa70-4394ccb239a0)

Jul 29 (single spike)

![image](https://github.com/user-attachments/assets/efd8e736-068c-4a1f-8ae0-b807e3578709)